packages/react/src/components/TimePicker/TimePicker.test.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { TimePicker } from './TimePicker';
+import { ITimeRange } from './TimePicker.types';
+import { create } from '@fluentui/utilities/lib/test';
+import { mount } from 'enzyme';
+import { IComboBox } from '../ComboBox/ComboBox.types';
+
+describe('TimePicker', () => {
+  it('renders correctly', () => {
+    const timeRange: ITimeRange = {
+      start: 0,
+      end: 5,
+    };
+    const component = create(<TimePicker label="I am a TimePicker" timeRange={timeRange} />);
```

Comment:
For some reason the initially selected time shown in this snapshot changed in the CI build, [causing a failure](https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=208065&view=logs&j=258ec178-2d8b-5611-7b9b-60c5c95dae55&t=a347995d-6693-5610-51bd-74417b4bee80&l=14730). I disabled this test for now, but later you'll need to update it to ensure it's not dependent on the locale or time zone of the machine, or the time the test is being run (I'm not totally sure what happened in this case).